nixos/modules/services/x11/display-managers/gdm.nix

```diff
@@ -133,15 +133,15 @@ in
         password required       pam_deny.so
 
         session  required       pam_succeed_if.so audit quiet_success user = gdm
-        session  required       pam_env.so envfile=${config.system.build.pamEnvironment}
+        session  required       pam_env.so readenv=0 conffile=${config.system.build.pamEnvironment}
```

Comment:
Since the default of readenv is 1, perhaps people are relying on .pam_environment being read by PAM. Is there a reason these changes are part of this PR?